test-suite/src/aggregate/error.rs

```diff
@@ -1,6 +1,8 @@
+use gluesql_core::executor::EvaluateError;
```

Comment:
Combine it with the import statement below.

Reply:
oh i forgot it thanks!